openwisp_users/admin.py

```diff
@@ -74,6 +74,10 @@ def _construct_form(self, i, **kwargs):
         return form
 
 
+class OrganizationOwnerInline(admin.StackedInline):
+    model = OrganizationOwner
```

Comment:
add `extra = 0`

Reply:
also here we should manipulate the permissions:

- add/change/delete: only owners and superusers

Reply:
@nemesisdesign I don't think we should add this here because the default permissions allow only superusers to see the `Organization` model. all other users can't even see it. But i think #143 will solve and meet this need

Reply:
yes you're right